doc/new_user.rst

```diff
@@ -4,7 +4,11 @@ New user basics
 Set up teuthology
 -----------------
 
-#. Log in to teuthology.front.sepia.ceph.com
+#. Request access via the process described at:
+   https://github.com/ceph/sepia/blob/master/doc/adding_users.rst
```

Comment:
Ah bummer, maybe at some point a refactoring will be needed. My comment should not stop this from moving forward.